src/notebooks_service.py

```diff
@@ -95,10 +97,28 @@ def health():
 @authenticated
 def whoami(user):
     """Return information about the authenticated user."""
-    return Response(
-        json.dumps(user, indent=1, sort_keys=True),
-        mimetype='application/json',
+    headers = {auth.auth_header_name: 'token {0}'.format(auth.api_token)}
+    info = json.loads(
+        requests.request(
+            'GET',
+            '{prefix}/users/{user[name]}'.format(
+                prefix=auth.api_url, user=user
+            ),
+            headers=headers
+        ).text
     )
+    return jsonify(info)
+
+
+# @authenticated
+@app.route(urljoin(SERVICE_PREFIX, 'ui'), defaults={'path': ''})
+@app.route(urljoin(SERVICE_PREFIX, 'ui/'), defaults={'path': ''})
```

Comment:
is this necessary? I didn't find that I needed to specifically include routes with trailing slashes

Reply:
You are right, it is unnecessary. I have removed the route without the trailing slash, and it still works when I provide the URL without the trailing slash.